### ISSUE TITLE
Fix import of compounds from KNOWNS.csv file.

### ISSUE THREAD

**Issue by PMSeitzer** [eugenemel/maven#63]
Broken in `8.0.23`, Identified by @eugenemel in `8.1.9`

**Comment by PMSeitzer** [comment]
Hey @eugenemel , Would you mind providing some information about the bug? When I try to update this using the latest maven, I didn't encounter any problems.

**Comment by eugenemel** [comment]
Hi. I figured it out. The schema for the ligand.db stored in the personal folder ~/Library/Application Support/Maven2/ligand.db was old. I deleted the ligand.db file, and not able to import KNOWNS.csv.   I think the fix here is to alter compound table to add adduct column if it doesn't, although sqlite might not support this.. need to check

**Comment by PMSeitzer** [comment]
OK, I can make the adjustment. Should just be 
```
ALTER TABLE compounds ADD new_column_name column_definition;
```

**Comment by PMSeitzer** [comment]
This may be fixed with `28d43d5298e584f3463d48021e2b45fdaf6f1842`, need to check
